.gitignore

```diff
@@ -60,4 +60,4 @@ testem.log
 libs/remix-node/
 libs/remix-niks/
 
-apps/remix-react
\ No newline at end of file
+apps/remix-react
```

Comment:
sorry I added those test ignores to the file, these can be removed
libs/remix-node/
libs/remix-niks/

apps/remix-react

Reply:
ok